converter/src/main/java/gov/cms/qpp/conversion/validate/CpcMeasureDataValidator.java

```diff
@@ -64,14 +65,24 @@ private void validateAllSupplementalNodesOfSpecifiedType(
 		EnumSet<SupplementalData> codes = EnumSet.copyOf(
 				 SupplementalData.getSupplementalDataSetByType(supplementalDataType));
 
-		for (SupplementalData supplementalData : codes) {
-			Node validatedSupplementalNode = filterCorrectNode(supplementalDataNodes, supplementalData);
+		String measureId = node.getParent().getValue(QualityMeasureIdValidator.MEASURE_ID);
+		MeasureConfig config = MeasureConfigs.getConfigurationMap().get(measureId);
+		if (config != null) {
+			String electronicMeasureID = config.getElectronicMeasureId();
+			for (SupplementalData supplementalData : codes) {
+				Node validatedSupplementalNode = filterCorrectNode(supplementalDataNodes, supplementalData);
 
-			if (validatedSupplementalNode == null) {
-				addSupplementalValidationError(node, supplementalData);
+				if (validatedSupplementalNode == null) {
+					addSupplementalValidationError(node, supplementalData, electronicMeasureID);
+				} else {
+					LocalizedError error = makeIncorrectCountSizeLocalizedError(node, supplementalData.getCode(),
+						electronicMeasureID);
+					check(validatedSupplementalNode)
+						.childMinimum(error, 1, TemplateId.ACI_AGGREGATE_COUNT)
+						.childMaximum(error, 1, TemplateId.ACI_AGGREGATE_COUNT);
```

Comment:
I've seen this in multiple places... do we not have a validation for asserting that there is a specific count of a `Node`?  Rather than always pairing up `childMinimum` and `childMaximum`?

Reply:
I don't know for sure because I just moved this code. I will double check this. Maybe this is a good chance to refactor `Checker.java`

Reply:
Unfortunately, the checker does not have a check to ensure the exact number of children. New checker method for `childExact` incoming.

Reply:
A new story has been made to create a new checker method and refactor all usages of childMin + childMax to use the new method instead.